From 8b396e3c58284ee31a5649e62cb9645c5079135b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Thu, 16 Oct 2014 19:16:00 -0500
Subject: [PATCH] units: remove /dev/log to always create symlink

When upgrading from older systemds, /dev/log would
exist and we wouldn't be able to create the symlink successfully.

https://bugzilla.redhat.com/show_bug.cgi?id=1099299
---
 units/systemd-journald-dev-log.socket | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/units/systemd-journald-dev-log.socket b/units/systemd-journald-dev-log.socket
index ffd44bb507..0950424163 100644
--- a/units/systemd-journald-dev-log.socket
+++ b/units/systemd-journald-dev-log.socket
@@ -17,6 +17,8 @@ Before=sockets.target
 IgnoreOnIsolate=yes
 
 [Socket]
+ExecStartPre=-/bin/rm -f /dev/log
+
 Service=systemd-journald.service
 ListenDatagram=/run/systemd/journal/dev-log
 Symlinks=/dev/log
